src/test/config.sol

```diff
@@ -333,12 +333,12 @@ contract Config {
             flip_ttl:     0,
             flip_tau:     0,
             flipper_mom:  0,
-            dog_hole:     40 * MILLION,
-            clip_buf:     12000,
-            clip_tail:    140 minutes,
-            clip_cusp:    4000,
+            dog_hole:     30 * MILLION,
+            clip_buf:     110_00,
+            clip_tail:    7200 seconds,
```

Comment:
```suggestion
            clip_tail:    7200,
```

Looks like this happened when `minutes` got converted, which is just a shorthand for `x * 60`, but let's keep to less code